app/src/main/java/com/nervousfish/nervousfish/activities/LoginActivity.java

```diff
@@ -11,25 +11,29 @@
 import com.nervousfish.nervousfish.R;
 import com.nervousfish.nervousfish.service_locator.IServiceLocator;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * Demo Login activity class. All warnings are suppressed because this is a demo class that can be deleted
  * after we're comfortable writing Cucumber tests.
  */
 @SuppressWarnings("PMD")
 public final class LoginActivity extends Activity {
-
+    private static final Logger LOGGER = LoggerFactory.getLogger("LoginActivity");
     private EditText mPassword;
     private View mError;
     private IServiceLocator serviceLocator;
 
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        setContentView(R.layout.login);
+        LOGGER.info("LoginActivity entered");
```

Comment:
Maybe change this to `LoginActivity created` and move it to the end of the method for clarity